build/source/netcdf/read_icond.f90

```diff
@@ -185,71 +186,79 @@ subroutine read_icond(iconFile,                      & ! name of initial conditi
  err = nf90_inq_dimid(ncID,"hru",dimID);               if(err/=nf90_noerr)then; message=trim(message)//'problem finding hru dimension/'//trim(nf90_strerror(err)); return; end if
  err = nf90_inquire_dimension(ncID,dimID,len=fileHRU); if(err/=nf90_noerr)then; message=trim(message)//'problem reading hru dimension/'//trim(nf90_strerror(err)); return; end if
 
- ! is there any snow in initial conditions file?
```

Comment:
There are no variables specific to snow: The state variables in the snow+soil domain (temperature, ice content liquid content) are defined for all layers. The only exception is matric head, which is defined only for soil layers.